addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlUtils.java

```diff
@@ -139,4 +142,95 @@ public static List<Map<String, String>> confirmUsableParameters(
         }
         return paramGroups;
     }
+
+    /**
+     * Removes all html tags from a given content string.
+     *
+     * @param str the content string to be stripped.
+     * @return a string without html tags.
+     */
+    public static String removeTags(String str) {
+        return str.replaceAll("(?s)<.*?>", "");
+    }
+
+    /**
+     * Returns a list of lines that are comon between two content Strings.
+     *
+     * @param body1 the first content string.
+     * @param body2 the second content string.
+     * @return a list of lines that are comon between two content Strings.
+     */
+    public static List<String> diffMap(String body1, String body2) {
+        List<String> diff = new ArrayList<>();
+        String[] body1Lines = StringUtils.split(body1, "\n");
+        String[] body2Lines = StringUtils.split(body2, "\n");
+        for (int i = 0; i < body1Lines.length; i++) {
+            if (body1Lines[i].equalsIgnoreCase(body2Lines[i])) {
+                diff.add(StringUtils.strip(body1Lines[i]));
+            }
+        }
+        return diff;
+    }
```

Comment:
By chance have you tested this on something that just has single  dumb long strings with no CR or LF?